### PR TITLE
feat: using canvas

### PR DESCRIPTION
**Pull request by petrSchreiber**

### CODE REVIEW

File: poc/Cornelius.tbasic [hunk 1]
```diff
@@ -707,27 +710,28 @@ function sleepDisplay()
...
+  While IsWindow(hWnd)
     call drawing
-' -- ESCAPE key to exit application
-    If tbgl_GetWindowKeyState(hWnd, %VK_ESCAPE) Then Exit While
+    
+    if asc(Canvas_Inkey)=  27 Then Exit While
   Wend
 
-  tbgl_DestroyWindow
+  Canvas_BitmapSaveImageFile("result.png")
```
Comment (petrSchreiber) [inline poc/Cornelius.tbasic:719]: The output looks correct to me:
result

Comment (DirectuX) [reply poc/Cornelius.tbasic:719]: This is beyond expectation ! Thanks again !

**Comment by DirectuX** [comment]
Hi Petr, 
I ran a stress-test on Cornelius ( maxHalfWidth = 10 and a 9 steps E S W N loop )
it took 1h16min and the result is below. 

<details>
<summary> Show picture </summary>

stresstest - 1H16
</details>

as you can see there is an inexplicable glitch in the upper right corner.
However, this "insane" map gave me issues to think of, from gameplay/story points of view essentially. I'll try to summarise them soon in the [projects](https://github.com/petrSchreiber/BetweenTheBubbles-tb/projects)

**Comment by petrSchreiber** [comment]
Wow,

this is so fascinating to watch 🥇.

I see the glitch and do not understand it, looking forward to further comments.

As for the 1h+ time. We currently approach the generation "per pixel". If we took geometric approach (working with line segments, with width as metadata), maybe we could be more time-efficient in lookups, for example. But I am not 100% sure.

To give you idea:
- now, we check isGameZone per pixel
- with geometric approach, we would just check, if given point is on line of given width (faster - possibly?)

**Comment by petrSchreiber** [comment]
@DirectuX this particular branch, using canvas, is just about rendering via Canvas.

While we need to further answer the glitch you discovered, it is related to algorithm, so we could maybe merge this to feat_proceduralMap and continue there?

**Comment by DirectuX** [comment]
Yes Petr, we should keep the last iteration of this Poc : the one with canvas

**Pull request merged**